app/models/topic.rb

```diff
@@ -0,0 +1,12 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id         :bigint           not null, primary key
+#  name       :string
+#  image      :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Topic < ApplicationRecord
```

Comment:
What if you add a validation to check `name` presence? Also add it to the spec file.

Reply:
Its done, thanks